week-3/Homework/mandatory/3-space-colonies.js

```diff
@@ -13,7 +13,13 @@
   
 */
 
-function colonisers() {}
+function colonisers(voyagers) {
+  let checkFamily = voyagers.filter((i) => i.includes("family"));
+
+  let startA = checkFamily.filter((i) => i.charAt(0) === "A");
+  return startA;
+}
+console.log(colonisers);
```

Comment:
Again, there is a shorter way of writing this function. See the solution here https://github.com/anthonytranDev/week-3-solutions/blob/master/mandatory/3-space-colonies.js